.github/workflows/manual-deploy-to-gh-pages.yml

```diff
@@ -1,4 +1,4 @@
-name: Deploy manually to gh-pages
+name: Docs deploy latest
```

Comment:
If we are renaming anyway, how do you like this:
- npm release
- docs deploy
- legacy npm release
- legacy docs deploy
- AUTO npm release and docs deploy
- AUTO Visual Regression Test

This way it is always prefixed and grouped by the category

Reply:
More consistent, I like it